### PR TITLE
CFE-3405 cf-remote: Added deploy command for deploying masterfiles

### PR DESCRIPTION
**Pull request by olehermanse**
```
olehermanse@OH-WIN core $ cf-remote deploy --hub hub /northern.tech/cfengine/masterfiles
ubuntu@34.250.47.156
OS            : ubuntu (debian)
Architecture  : x86_64
CFEngine      : 3.17.0a.ec47889f4 (Enterprise)
Policy server : None
Binaries      : dpkg, apt
Copying: '/home/olehermanse/.cfengine/cf-remote/masterfiles.tgz' to 'ubuntu@34.250.47.156'
Running: 'systemctl stop cfengine3 && rm -rf /var/cfengine/masterfiles && mv masterfiles /var/cfengine/masterfiles && systemctl start cfengine3 && cf-agent -Kf update.cf && cf-agent -K'
```

### CODE REVIEW

File: contrib/cf-remote/cf_remote/main.py [hunk 1]
```diff
@@ -108,34 +108,32 @@ def run_command_with_args(command, args):
...
+        return commands.spawn(args.platform, args.count, args.role, args.name)
     elif command == "destroy":
         group_name = args.name if args.name else None
-        commands.destroy(group_name)
+        return commands.destroy(group_name)
     elif command == "deploy":
-        commands.deploy(args.hub, args.directory)
+        return commands.deploy(args.hub, args.directory)
     else:
         user_error("Unknown command: '{}'".format(command))
```
Comment (vpodzime) [inline contrib/cf-remote/cf_remote/main.py:138]: missing `return 1` here

**Review by vpodzime — COMMENTED** [review]
Looks good to me otherwise.

File: contrib/cf-remote/cf_remote/main.py (hunk 1 above)
Comment (olehermanse) [reply contrib/cf-remote/cf_remote/main.py:138]: not really, `user_error` is `sys.exit`

Comment (vpodzime) [reply contrib/cf-remote/cf_remote/main.py:138]: Then it's another `sys.exit()` in place where `return` should be used. The code calling this function might be doing some cleanup before exiting, for example.

**Review by vpodzime — APPROVED** [review]
ACK

**Pull request merged**